tests/net/socket/socketpair/src/test_socketpair_happy_path.c

```diff
@@ -115,6 +115,7 @@ static void happy_path(
 
 		memset(actual_msg, 0, sizeof(actual_msg));
 
+		len = 0;
```

Comment:
A better fix would be to initialize the variable at its declaration up on line 48, as Coverity notes:
```
   	14. var_decl: Declaring variable len without initializer.
 48        socklen_t len;
```

Reply:
Ah, ok. It is a loop though, so it should be initialized twice (once for each of the two iterations) before passing by reference.

Reply:
No; it's actually not used at all, because `src_addr` is NULL.  But Coverity doesn't know that, so we need to give it a value just once.

Reply:
Except Coverity doesn't get that 😭 You & I do, of course. Reinitializing `len` before the function is called is technically more correct-ish, should mitigate the Coverity issue, and that's all I really care about.